protocol/x/assets/types/codec.go

```diff
@@ -3,21 +3,12 @@ package types
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
-	// this line is used by starport scaffolding # 1
+
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
 )
 
-func RegisterCodec(cdc *codec.LegacyAmino) {
-	// this line is used by starport scaffolding # 2
-}
+func RegisterCodec(cdc *codec.LegacyAmino) {}
 
 func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
-	// this line is used by starport scaffolding # 3
-
 	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
 }
-
-var (
```

Comment:
Hmm, this might be required when trying to use `x/authz` later (which we prev discussed might be of use). So I don't think we should remove these just yet

See [example](https://github.com/osmosis-labs/osmosis/blob/013cd96fdd40d0ddcff3351240492bbc3a3c10ed/x/lockup/types/codec.go#L38).

Reply:
Hmm, trying to dig into this. I don't currently understand where Amino is or is not used. Seems like it isn't used in our current application, but that certain modules might still rely on their amino codec being configured?

Reply:
Yah, I am not 100% sure if Amino is fully unused either. Based on this comment [here](https://github.com/dydxprotocol/v4-chain/blob/68cea103835050b218a3672d02bf6389c331e23f/protocol/lib/encoding/encoding.go#L29), there might still be some use case. 

This feels like a P3 issue/cleanup task that can be deferred to M6+. Can we just create a ticket to track this rather than spending more time on this now?

Reply:
cut an issue to track https://linear.app/dydx/issue/CORE-556/determine-if-legacy-amino-codec-is-required-for-xmodules